client/app/application.coffee

```diff
@@ -47,34 +46,49 @@ class Application extends Mn.Application
 
         # render components when app starts
         @on 'start', ->
+            @layout.render()
+
             @contacts.fetch reset: true
             @tags.fetch reset: true
 
-            @layout.render()
-
             # prohibit pushState because URIs mapped from cozy-home rely on
             # fragment
             Backbone.history.start pushState: false if Backbone.history
 
 
+    getSelected: ->
+        view = @layout.getChildView 'content'
+        view.children.reduce (memo, view) ->
```

Comment:
I find using a reduce for this case less readable than a simple
```coffee
selected = []
for view in view.children
   if ...
return selected
``` 
or even better, with an helper function 
```coffee
getSubViews().map (view) -> view.model.model
```